### PR TITLE
Fixes #489: remove outdated Uphold tests on Android

### LINKED ISSUE #489: Remove minimum-BAT (Uphold) test on Android
We can remove this test, as Uphold no longer specifies a minimum BAT amount:

>  Verify on BR panel that the Verify wallet button only enables when balance is >=15 BAT

https://github.com/brave/qa-resources/blob/b484d646bee441a5af5c44704d3907e5e2c0ebff/WikiTemplate/Android/wikitemplate-ReducedAndroid.md#rewards

https://github.com/brave/qa-resources/blob/b484d646bee441a5af5c44704d3907e5e2c0ebff/WikiTemplate/Android/wikitemplate-android.md#brave-rewardsads

cc @LaurenWags @srirambv @kjozwiak

### PR DESCRIPTION
**Pull request by stephendonner**
LMK if you want me to include the `- [ ] Verify user still able to connect to existing Uphold account via panel even when balance is < 2 BAT` test too

### CODE REVIEW

**Comment by LaurenWags** [comment]
IMO - remove `- [ ] Verify user still able to connect to existing Uphold account via panel even when balance is < 2 BAT` also. Basically remove anything referencing limits.

However, I do think we should have some kind of generic "Verify able to link KYC'd Uphold account on a clean profile" or some similar case. Maybe `Verify you are able to disconnect/re-connect a user wallet` covers that since you must connect in order to disconnect?

Thoughts?

**Comment by stephendonner** [comment]
@LaurenWags good suggestions; I changed it to `- [ ] Verify you are able to connect, disconnect, and re-connect a KYC'd (Uphold, etc.) user wallet` - how does that look?

File: WikiTemplate/Android/wikitemplate-android.md [hunk 1]
```diff
@@ -96,9 +96,7 @@
 - [ ] Verify unchecking `Allow contribution to videos` option doesn't list any YouTube creator in ac list
 - [ ] Adjust min visit/time in settings. Visit some sites and YouTube channels to verify they are added to the table after the specified settings
 - [ ] Verify you can reset rewards from advance setting. Resetting should delete wallet and bring it back to the pre-optin state
-- [ ] Verify on BR panel `Verify wallet` button loads verify wallet page when balance in >=2 BAT.
-- [ ] Verify user still able to connect to existing Uphold account via panel even when balance is < 2 BAT
-- [ ] Verify you are able to disconnect/re-connect a user wallet
+- [ ] Verify you are able to connect, disconnect, and re-connect a KYC'd (Uphold, etc.) user wallet
```
Comment (srirambv) [inline WikiTemplate/Android/wikitemplate-android.md:99]: Maybe change to
`Verify you are able to connect, disconnect, and re-connect a KYC'd user wallet (Uphold, Gemini etc.)`

Comment (stephendonner) [reply WikiTemplate/Android/wikitemplate-android.md]: fixed: https://github.com/brave/qa-resources/commit/13dd165e484ee27379c257791ea2f729239b7b98

**Review by LaurenWags — APPROVED** [review]
++ lgtm, thanks for making these updates!

**Comment by stephendonner** [comment]
Thanks for the review & feedback, @LaurenWags and @srirambv ❤️

**Pull request merged**